scss/bitstyles/components/dropdown-menu.stories.mdx

```diff
@@ -0,0 +1,161 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Components/Dropdown menu" />
+
+# Navbar
```

Comment:
Maybe I have the wrong concept of a navbar, but I'm more thinking about something like [this](https://getbootstrap.com/docs/4.0/components/navbar/) when I hear that term.

Reply:
Arg, I’ll check this — looks like a copy/paste error! This is clearly taken from the navbar component 🙈 Thanks!

Reply:
Updated! Please take another look if you have time 🙏